materials/data/Concrete_Broomfinish_permittivity.gltf

```diff
@@ -0,0 +1,44 @@
+{
+    "asset": {
+        "copyright": "(C) 2021, Bayerische Motoren Werke Aktiengesellschaft (BMW AG)",
+        "generator": "hand-made",
+        "version": "1.0",
+        "extensions": {
+            "OpenMaterial_asset_info": {
+                "asset_type": "material_permittivity",
+                "id": "...",
+                "title": "relative permittivity Concrete_BF",
+                "asset_version": "1",
+                "asset_variation": "1",
+                "creator": "Bayerische Motoren Werke Aktiengesellschaft (BMW AG), Technische Universitaet Muenchen - Professur fuer Hoechstfrequenztechnik",
+                "description": "relative permittivity of conrete roads built in using broomfinish technique  (Concrete_BF)",
+                "creation_date": "2022.01.10. 12:00:00",
+                "sources": [
+                    "https://ars.copernicus.org/articles/19/165/2021/"
+                ],
+                "comment": "relative permittivity of conrete roads built in using broomfinish technique (Concrete_BF)"
+            }
+        }
+    },
+    "extensions": {
+        "OpenMaterial_permittivity_data": {
+            "data": [
+                {
+                    "temperature": 280.0,
+                    "incident_angle": 0.78,
+					"frequency_range": [
```

Comment:
This element is not part of the specification (see https://github.com/LudwigFriedmann/OpenMaterial/tree/issue_10_radar_materials_permeability_permittivity/glTF_extensions/OpenMaterial_permittivity_data). Please remove it.